src/services/authService.js

```diff
@@ -41,6 +42,19 @@ class AuthService {
   getCurrentUser() {
     return JSON.parse(localStorage.getItem('user'));
   }
+
+  loanSignup(max_amount, name, interest_rate, balance, start_date) {
+    return axios.
+    post(API_URL2 + "loansignup?userId=" + this.getCurrentUser().id
+    , {
+      max_amount,
+      name,
+      interest_rate,
+      balance,
+      start_date
+    })
+  }
```

Comment:
You're passing the interest rate and balance of the loan to the back end...

...nooooooo...

Reply:
Again, this is the same issue you brought up in the backend MS, which I explained there...